units/crons.go

```diff
@@ -320,6 +322,34 @@ func PopulateLastContainerFinishTimeJobs() amboy.QueueOperation {
 	}
 }
 
+func PopulateParentDecommissionJobs() amboy.QueueOperation {
+	return func(queue amboy.Queue) error {
+		catcher := grip.NewBasicCatcher()
+		ts := util.RoundPartOfHour(1).Format(tsFormat)
+
+		// Find all active distros supporting containers
+		query := db.Query(bson.M{
```

Comment:
can you move this query into the model package?

It'll make more sense there, and also there's an informalish rule that the evergreen/db and bson packages should only be dependencies of the models package.

Reply:
Moving it for now, but this query and everything else using MaxContainers in the distro model will be refactored in the next ticket I'm working on.